app/components/provider_interface/organisation_permissions_form_component.rb

```diff
@@ -0,0 +1,82 @@
+module ProviderInterface
+  class OrganisationPermissionsFormComponent < ViewComponent::Base
+    attr_reader :provider_user, :provider_relationship_permission, :mode, :form_url
+
+    def initialize(provider_user:, provider_relationship_permission:, mode:, form_url:)
+      @provider_user = provider_user
+      @provider_relationship_permission = provider_relationship_permission
+      @mode = mode
+      @form_url = form_url
+    end
+
+    def page_caption
+      if mode == :edit
+        provider_relationship_description
+      elsif mode == :setup
+        t('.setup.caption')
+      end
+    end
+
+    def page_heading
+      if mode == :edit
+        t('.edit.heading')
+      elsif mode == :setup
+        provider_relationship_description
+      end
+    end
+
+    def checkbox_details_for_providers
+      ordered_providers.map do |provider_type|
+        {
+          name: name_for_provider_of_type(provider_type),
+          type: provider_type,
+        }
+      end
+    end
+
+    def label_for(permission_name)
+      "Who can #{t("provider_relationship_permissions.#{permission_name}.description").downcase}?"
+    end
```

Comment:
Should this be a separate locale entry, maybe interpolate one with the lowercase output from the other?

Reply:
Yeah that's a good idea 👍